RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -32,7 +34,8 @@
         'detachedTripletStepTracks',
         'mixedTripletStepTracks',
         'pixelLessStepTracks',
-        'tobTecStepTracks'
+        'tobTecStepTracks',
+        'siStripTripletStepTracks'
```

Comment:
are we sure ?
when such a modifier is run ? in which workflows ?
I'm not super sure we want to run this new step here ....